packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -72,32 +69,41 @@ const addProcessEnv = async (log, env) => {
 };
 
 const deploymentErrorMsg = `Your deployment failed. Please retry later. More: https://err.sh/now/deployment-error`;
-const prepareAlias = input => isWildcardAlias(input) ? input : `https://${input}`;
+const prepareAlias = input =>
+  isWildcardAlias(input) ? input : `https://${input}`;
 
 const printDeploymentStatus = async (
   output,
-  { url, readyState, alias: aliasList, aliasError },
+  { readyState, alias: aliasList, aliasError },
   deployStamp,
   clipboardEnabled,
   localConfig,
   builds
 ) => {
-  if (readyState === 'READY') {
-    if (aliasError && aliasError.message) {
-      output.warn(`Failed to assign aliases: ${aliasError.message}`);
-    }
+  if (aliasError && aliasError.message) {
+    output.warn(`Failed to assign aliases: ${aliasError.message}`);
+  }
```

Comment:
We should only display this when the deployment is `READY`, since I think it could indicate that the deployment failed because of this, which would not be the case.